docarray/documents/video.py

```diff
@@ -97,12 +99,30 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[VideoUrl]
-    audio: Optional[AudioDoc] = AudioDoc()
-    tensor: Optional[VideoTensor]
-    key_frame_indices: Optional[AnyTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[VideoBytes]
+    url: Optional[VideoUrl] = Field(
+        description='URL to the remote video file that needs to be loaded',
+        example='https://github.com/docarray/docarray/blob/main/tests/toydata/mov_bbb.mp4?raw=true',
+    )
+    audio: Optional[AudioDoc] = Field(
+        description='Audio document document associated with the video',
+        default=AudioDoc(),
+    )
+    tensor: Optional[VideoTensor] = Field(
+        description='''Tensor object of the video which be specified to one of
+        `VideoNdArray`, `VideoTorchTensor`, `VideoTensorFlowTensor`''',
```

Comment:
Skipping example here since we will need to do load tensor from video url and needs a multi-line example